contracts/token/ERC20/StandardToken.sol

```diff
@@ -156,4 +156,41 @@ contract StandardToken is ERC20 {
     return true;
   }
 
+  /**
+   * @dev Internal function that mints an amount of the token and assigns it to
+   * an account. This encapsulates the modification of balances such that the
+   * proper events are emitted.
+   * @param _account The account that will receive the created tokens.
+   * @param _amount The amount that will be created.
+   */
+  function _mint(address _account, uint256 _amount) internal {
+    totalSupply_ = totalSupply_.add(_amount);
```

Comment:
Do we want to do input validation on `amount > 0` and `account != 0`?

Reply:
`_amount > 0` is a validation that we have decided in the past to not do, because it forces users of the function to special-case `amount == 0` to avoid calling this function.

`_account != 0` I guess we should add.